<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela_produto</class>
 <widget class="QDialog" name="tela_produto">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>467</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>846</width>
    <height>265</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Produto</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="gb_dados_produto">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>55</y>
     <width>836</width>
     <height>206</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Dados do produto</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="lb_nome">
    <property name="geometry">
     <rect>
      <x>258</x>
      <y>20</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Nome:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_nome">
    <property name="geometry">
     <rect>
      <x>293</x>
      <y>20</y>
      <width>538</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>60</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_fabricante">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>50</y>
      <width>347</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>60</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_fabricante">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>50</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Fabricante:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_codigo_barras">
    <property name="geometry">
     <rect>
      <x>422</x>
      <y>50</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Código de Barras:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_codigo_barras">
    <property name="geometry">
     <rect>
      <x>521</x>
      <y>50</y>
      <width>105</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="inputMask">
     <string>0 000000 000000; </string>
    </property>
    <property name="text">
     <string>  </string>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_quantidade">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>80</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="inputMask">
     <string>0000000000; </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_quantidade">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>80</y>
      <width>67</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Quantidade:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_des_utilizacao">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>128</y>
      <width>621</width>
      <height>72</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Calibri'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_comentario">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>108</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Descrição da utilização:</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gv_imagem_produto">
    <property name="geometry">
     <rect>
      <x>631</x>
      <y>50</y>
      <width>200</width>
      <height>150</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="lb_tipo">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>20</y>
      <width>28</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Tipo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_valorvenda">
    <property name="geometry">
     <rect>
      <x>388</x>
      <y>80</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Valor de venda atual:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_valor_compra">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>80</y>
      <width>127</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Valor de compra atual:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_valor_venda">
    <property name="geometry">
     <rect>
      <x>508</x>
      <y>80</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string>R$ xxxxxxxxx; </string>
    </property>
    <property name="text">
     <string>R$ </string>
    </property>
    <property name="maxLength">
     <number>12</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_valor_compra">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string>R$ xxxxxxxxx; </string>
    </property>
    <property name="text">
     <string>R$ </string>
    </property>
    <property name="maxLength">
     <number>12</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_tipo">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>20</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_codigo">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>20</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="inputMask">
     <string>000000000; </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="cursorPosition">
     <number>9</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_codigo">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>20</y>
      <width>42</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Código:</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>846</width>
     <height>50</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QToolButton" name="btn_editar_produto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>2</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Editar Produto</string>
    </property>
    <property name="icon">
     <iconset resource="imagens.qrc">
      <normaloff>:/img/img/produto_editar.png</normaloff>:/img/img/produto_editar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_adicionar_produto">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>2</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Adiconar Produto</string>
    </property>
    <property name="icon">
     <iconset resource="imagens.qrc">
      <normaloff>:/img/img/produto_cadastrar.png</normaloff>:/img/img/produto_cadastrar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_remover_produto">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>2</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Remover Produto</string>
    </property>
    <property name="icon">
     <iconset resource="imagens.qrc">
      <normaloff>:/img/img/produto_remover.png</normaloff>:/img/img/produto_remover.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonIconOnly</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tw_historico_estoque">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>270</y>
     <width>316</width>
     <height>192</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="imagens.qrc"/>
 </resources>
 <connections/>
</ui>
